tests/integration_tests/test_command_line_options.py

```diff
@@ -28,21 +28,21 @@ def test_custom_options(self, fake_template_doer):
             "-td",
             "/home/developer/templates",
             "-t",
-            "a.template",
+            "a.jj2",
```

Comment:
do we still have a test of `.template` to ensure testing of unknown extensions which should default to jinja

Reply:
Yes, unknown file extension will be rendered by jinjia2

Reply:
with my additional changes, the condition is: user have not changed template_type to other type, jinja will be the default engine.